docs/cli-ref.rst

```diff
@@ -35,3 +35,46 @@ development and testing.
    :module: airflow.cli.cli_parser
    :func: get_parser
    :prog: airflow
+
+Environment Variables
+'''''''''''''''''''''
+
+.. envvar:: AIRFLOW__{SECTION}__{KEY}
+
+  Any specific key in a section in Airflow.
+  The user may replace the ``{SECTION}`` placeholder with any section
+  and the ``{KEY}`` placeholder with any key in that specified section.
+  For example, the user may initiate "AIRFLOW__CORE__DAGS_FOLDER",
+  where CORE is the section and DAGS_FOLDER is the key within the section.
+
+  For more information, see: :doc:`/howto/set-config`.
+
+.. envvar:: AIRFLOW__{SECTION}__{KEY}_CMD
+
+  For any specific key in a section in Airflow,
+  execute the command the key is pointing to.
+
+.. envvar:: AIRFLOW_CONFIG
+
+  The path to the Airflow configuration file.
+
+.. envvar:: AIRFLOW_CONN_{CONN_ID}
+
+  Defines a connection URI.
+  The user may replace the ``{CONN_ID}`` placeholder with the connection name.
+  For example, the user may create a connection named "AIRFLOW_CONN_PROXY_POSTGRES_TCP"
+  to connect via proxy over TCP.
+
+  For more information, see: :ref:`environment_variables_secrets_backend`.
+
+.. envvar:: AIRFLOW_HOME
+
+  The root directory for the Airflow content.
+  This is the default parent directory for Airflow assets such as DAGs and logs.
+
+.. envvar:: AIRFLOW_VAR_{KEY}
+
+  Defines an Airflow variable.
+  The user may replace the ``{KEY}`` placeholder with the variable name.
+
+  For more information, see: <link to Storing Variables in Environment Variables on https://airflow.readthedocs.io/en/latest/howto/variable.html>.
```

Comment:
Can you add use ref here? See: https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page

Reply:
@mik-laj yes 😄 this is done!